docs/framework/get-started/overview.md

```diff
@@ -20,85 +20,90 @@ author: "rpetrusha"
+
+While the runtime is designed for the software of the future, it also supports software of today and yesterday. Interoperability between managed and unmanaged code enables developers to continue to use necessary COM components and DLLs.
+
+The runtime is designed to enhance performance. Although the common language runtime provides many standard runtime services, managed code is never interpreted. A feature called just-in-time (JIT) compiling enables all managed code to run in the native machine language of the system on which it's executing. Meanwhile, the memory manager removes the possibilities of fragmented memory and increases memory locality-of-reference to further increase performance.
+
+Finally, the runtime can be hosted by high-performance, server-side apps, such as Microsoft SQL Server and Internet Information Services (IIS). This infrastructure enables you to use managed code to write your business logic, while still enjoying the superior performance of the industry's best enterprise servers that support runtime hosting.
+
+## .NET Framework class library
+
+The .NET Framework class library is a collection of reusable types that tightly integrate with the common language runtime. The class library is object oriented, providing types from which your own managed code derives functionality. This not only makes the .NET Framework types easy to use but also reduces the time associated with learning new features of the .NET Framework. In addition, third-party components integrate seamlessly with classes in the .NET Framework.
+
+For example, the .NET Framework collection classes implement a set of interfaces for developing your own collection classes. Your collection classes blend seamlessly with the classes in the .NET Framework.
+
+As you would expect from an object-oriented class library, the .NET Framework types enable you to accomplish a range of common programming tasks, including tasks such as string management, data collection, database connectivity, and file access. In addition to these common tasks, the class library includes types that support a variety of specialized development scenarios. Use the .NET Framework to develop the following types of apps and services:
+
+- Console apps. See [Building Console Applications](../../../docs/standard/building-console-apps.md).
+
+- Windows GUI apps (Windows Forms). See [Windows Forms](../../../docs/framework/winforms/index.md).
+
+- Windows Presentation Foundation (WPF) apps. See [Windows Presentation Foundation](../../../docs/framework/wpf/index.md).
+
+- ASP.NET apps. See [Web Applications with ASP.NET](../../../docs/framework/develop-web-apps-with-aspnet.md).
```

Comment:
Should this also list ASP.NET Core? Or does that fall under the ASP.NET umbrella?